react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+
+export default function Stopwatch() {
+  let [time, setTime] = useState(0);
+  let [running, setRunning] = useState(false);
+  const [icon, setIcon] = useState('play');
+
+  function resetFace() {
+    setTime(0);
+  }
+
+  function handleIconClick() {
+    clearInterval(running);
+    if (icon === 'play') {
+      setIcon('pause');
+      running = setInterval(() => {
+        setTime(time++);
+      }, 1000);
+    }
+    if (icon === 'pause') {
+      setIcon('play');
+      clearInterval(running);
+    }
+    setRunning(running);
```

Comment:
Since `running` is the interval you use to run the clock, it would be best to only set it when you call `setInterval` and then unset it when you call `clearInterval`.